server/lib/transferwise.ts

```diff
@@ -78,8 +80,9 @@ export const createQuote = async (
     });
     return getData(response);
   } catch (e) {
-    logger.error(`Unable to create quote: ${getErrorCode(e)}`, data);
-    throw new Error(`Sorry, we can't make transfers to ${targetCurrency}.`);
+    const error = parseError(e);
+    logger.error(error.toString(), data);
+    throw error;
```

Comment:
Have you considered some refactor to avoid repeating that everywhere and make the code more compact?

Reply:
Thanks, I refactored that file with a high order function to abstract the request/throw logic.